src/invert/fft_fftw.cxx

```diff
@@ -61,13 +61,47 @@ void fft_init(Options* options) {
   if (options == nullptr) {
     options = Options::getRoot()->getSection("fft");
   }
-  fft_init((*options)["fft_measure"]
-               .doc("Perform speed measurements to optimise settings?")
-               .withDefault(false));
+  bool fft_measure = (*options)["fft_measure"]
+                    .doc("Perform speed measurements to optimise settings?")
+                    .withDefault(false);
+  fft_flag = (*options)["fft_flag"]
+                    .doc("Level speed measurements to optimise FFT settings: [estimate], measure, exhaustive")
```

Comment:
Also might be worth noting that estimate is the only option certain to be exactly repeatable (as the other two make measurements and can choose different algorithms with each run).

Reply:
Good thing to note in the docs

Reply:
> Good thing to note in the docs

It already was - 39a00aeecaa1555be41e37de6aeb25b9319e64f1 by @ZedThree :smiley: